public/javascripts/components/conversation/conversation.js

```diff
@@ -78,7 +82,14 @@ window.ChubboChat.components.conversation = Vue.extend({
           text: this.chatInput,
           sender: 'user'
         });
+        this.surveyResponses.push('"' + this.chatInput + '"');
```

Comment:
probably best to still make this an object like
``` javascript
push({
  text: "w.e"
})
```

that way you could easily expand it here, adding attributes like `date`, `time`, `device` etc without breaking code elsewhere